azurerm/resource_arm_container_registry.go

```diff
@@ -57,6 +57,20 @@ func resourceArmContainerRegistry() *schema.Resource {
 				Default:  false,
 			},
 
+			"georeplication_enabled": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
+			},
+
+			"georeplication_locations": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
```

Comment:
You should use the ValidateFunc to verify the value passed.

Reply:
It seems that location does not get validation in the whole provider (see https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/azurerm/location.go). How would you validate that? Having a fixed list of region could introduce a blocker when new regions are available. And potentially, the location could be validated for all different services to make sure it is available in the region.

Reply:
Done - and ordering does not matter in that case.